content/docs/languages/go/quickstart.md

```diff
@@ -129,7 +129,7 @@ Before you can use the new service method, you need to recompile the updated
 While still in the `examples/helloworld` directory, run the following command:
 
 ```sh
-$ protoc -I helloworld/ helloworld/helloworld.proto --go_out=plugins=grpc:helloworld
+$ protoc -I helloworld --go_out=plugins=grpc:helloworld --go_opt=paths=source_relative helloworld.proto
```

Comment:
`helloworld.proto`?  This changed from `helloworld/helloworld.proto` -- is the path to the file no longer necessary because of the change to the `-I` or the `source_relative` option?

Reply:
Actually, the `-I` shouldn't be there at all, if this is running from `examples/helloworld` and the file's path is `examples/helloworld/helloworld/helloworld.proto`. The `-I` is only to bring in other protoc roots. (Basically, you just need `-I .`, but I think that is there by default.)

Reply:
Ok, here's what I've tried:

- Dropping the `-I` option or using `-I .` results in 

  ```nocode
  Could not make proto path relative: helloworld.proto: No such file or directory
  ```

- Dropping the `-I` option or using `-I .`, _and_ updating the source path to `helloworld/helloworld.proto` generates the `.pb.go` file under `examples/helloworld/helloworld/helloworld/helloworld.proto` (that's 3 `helloworld` directories, which isn't what we want). I guess that makes sense under `paths=source_relative`.

The following seems to work as desired:

```console
$ protoc --go_out=plugins=grpc:. --go_opt=paths=source_relative helloworld/helloworld.proto
```

This is what the PR currently contains:

```console
$ protoc -I helloworld --go_out=plugins=grpc:helloworld --go_opt=paths=source_relative helloworld.proto
```

Which command would you prefer that I use? (I'd vote for the former.)

Reply:
> Which command would you prefer that I use? (I'd vote for the former.)

I agree, that one seems to make more sense.

Reply:
Ok, I've updated the PR. PTAL